### PR TITLE
[com_users] - fix camel case

### PR DESCRIPTION
**Pull request by alikon**
Pull Request for Issue #17460 .

### Summary of Changes
The form field types are now camel case

`registrationdaterange vs RegistrationDateRange`

etc

### CODE REVIEW

File: administrator/components/com_users/models/forms/filter_users.xml [hunk 1]
```diff
@@ -37,7 +37,7 @@
 		</field>
 		<field
 			name="lastvisitrange"
-			type="lastvisitdaterange"
+			type="LastvisitDateRange"
```
Comment (brianteeman) [inline administrator/components/com_users/models/forms/filter_users.xml:40]: shouldnt this one be LastVisitDateRange if it is camelcase

Comment (alikon) [reply administrator/components/com_users/models/forms/filter_users.xml:40]: in theory yes, but in pratice  no
cause https://github.com/joomla/joomla-cms/blob/3.8-dev/libraries/src/Joomla/CMS/Form/Field/LastvisitDateRangeField.php#L37
`$this->type = 'LastvisitDateRange';`

maybe is _dromedary case_ :dromedary_camel:

**Comment by mbabker** [comment]
There shouldn't be a requirement for this.  So is this just for visual aesthetics or is there a legitimate issue?

File: administrator/components/com_users/models/forms/filter_users.xml (hunk 1 above)
Comment (brianteeman) [reply administrator/components/com_users/models/forms/filter_users.xml:40]: ok - just would seem correct to me for it to be corrected everywhere

**Comment by alikon** [comment]
for me  there is an issue see #17460 

if in the filter_users.xml for exmaple the `type="userstate" `  is all lower case
i was'nt able to to see the userstate filter populated

**Comment by mbabker** [comment]
Because of the class mapping stuff anything that was working fine on 3.7 should just work no issue on 3.8, hence the reason this should not be needed at all.

@laoneo thoughts?

**Comment by laoneo** [comment]
Yes, you are right @mbabker this shouldn't be a required change. I will have a look where the issue is in the next days. Looks like something else here is wrong.

**Comment by laoneo** [comment]
See my comment here https://github.com/joomla/joomla-cms/issues/17460#issuecomment-322424901. This change should not be required, otherwise we have a BC break.

**Comment by mbabker** [comment]
Closing as noted.

**Pull request closed without merging**